File: imageloader/src/main/java/com/sdwfqin/imageloader/progress/OnProgressListener.java
package com.sdwfqin.imageloader.progress;

/**
 * 描述：图片加载进度
 *
 * @author zhangqin
 * @date 2018/6/21
 */
public interface OnProgressListener {

    /**
     * 加载中
     * @param progress
     */
    void onLoading(int progress);

    /**
     * 加载成功
     */
    void onLoadSuccess();
}

File: imageloader/src/main/java/io/github/sdwfqin/imageloader/progress/OnProgressListener.kt
package io.github.sdwfqin.imageloader.progress

/**
 * 描述：图片加载进度
 *
 * @author zhangqin
 * @date 2018/6/21
 */
interface OnProgressListener {
    /**
     * 加载中
     * @param progress
     */
    fun onLoading(progress: Int)

    /**
     * 加载成功
     */
    fun onLoadSuccess()
}